statistics/feedback.go

```diff
@@ -811,24 +811,48 @@ func decodeFeedbackForPK(q *QueryFeedback, pb *queryFeedback, isUnsigned bool) {
 	}
 }
 
-func decodeFeedbackForColumn(q *QueryFeedback, pb *queryFeedback) error {
+// ConvertDatumsType converts the datums type to `ft`.
+func ConvertDatumsType(vals []types.Datum, ft *types.FieldType, loc *time.Location) error {
```

Comment:
Need we export it?

Reply:
Yes, it is also used in `convertRangeType`, which is in package `handle`.